bep032tools/generator/BEP032Generator.py

```diff
@@ -140,23 +140,32 @@ def basedir(self, basedir):
             raise ValueError('Base directory does not exist')
         self._basedir = Path(basedir)
 
-    def get_data_folder(self, mode='absolute'):
+    def get_data_folder(self, mode='absolute', ephys_type='extra'):
```

Comment:
To avoid confusion and be more consistent with NWB nomenclature we could use `ece` and `ice` here instead for `extracelluar electrophysiology` and `intracellular electrophysiology`